spec/controllers/admin/order_cycles_controller_spec.rb

```diff
@@ -45,15 +45,6 @@ module Admin
     end
 
     describe "new" do
-      describe "when the user manages no distributor enterprises suitable for coordinator" do
-        let!(:distributor) { create(:distributor_enterprise, owner: distributor_owner, confirmed_at: nil) }
-
-        it "redirects to order cycles index" do
-          spree_get :new
-          expect(response).to redirect_to admin_order_cycles_path
-        end
-      end
```

Comment:
We need to keep this spec, it is testing something important. We could use `sells: 'unconfirmed'` instead of `confirmed_at: nil`. That should have the same effect.

Reply:
@oeoeaio I don't think substituting `confirmat_at: nil` with `sells: 'unspecified'` is going to work every time. They're not completely equivalent. In this instance, the user has only one enterprise and they havn't set the "sells" property, which is part of the activation of their first enterprise. The redirect sends them to `/unauthorized` because they have to set it when they first log in, and can't see the rest of the admin section until they have an activated enterprise.

Reply:
Ok, but why is there a redirect to `/unauthorized`? Incomplete enterprise setup is not an auth issue. I feel like they should at least be redirected to the welcome page where they can set their "sells" property?

Reply:
I would argue that for subsequent enterprises with `sells: 'unspecified'` users should still be allowed to access the admin section, in which can this spec should still apply (though it may need tweaking).

Reply:
If the user has at least one enterprise activated, it allows them to directly access that page, but if they don't they get shown a page to activate their enterprise first. The menu isn't even visible to the user in this example, so they can't navigate to the order cycles page.